e2e/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:record": "cypress run --record"
+    "cypress:record": "cypress run --record",
+    "lint": "tslint -p tsconfig.json"
```

Comment:
Adds the `npm run lint` command to use both from the command line and from buildkite.